frontend/packages/dev-console/src/components/pipelines/pipeline-form/StartPipelineModal.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Formik, FormikValues } from 'formik';
+import {
+  createModalLauncher,
+  ModalComponentProps,
+} from '@console/internal/components/factory/modal';
+import { k8sCreate } from '@console/internal/module/k8s';
+import { PipelineRunModel } from '../../../models';
+import { Pipeline, PipelineResource, Param, PipelineRun } from '../../../utils/pipeline-augment';
+import StartPipelineForm from './StartPipelineForm';
+import { validationSchema } from './pipelineForm-validation-utils';
+
+export type newPipelineRun = (Pipeline: Pipeline, latestRun: PipelineRun) => {};
```

Comment:
Seems unnecessary to have a separate type as I couldn't find this being used elsewhere. Simply inline the definition below.
Can the return value be `K8sResourceKind` ?
```suggestion
export type newPipelineRun = (pipeline: Pipeline, latestRun: PipelineRun) => K8sResourceKind;
```